crates/frontend/src/pages/compare_overrides.rs

```diff
@@ -0,0 +1,218 @@
+                            context_vector.retain(|context| *context != remove_column_name)
+                    )
+                    >
+                    </i>
+                </span>
+            }.into_view()
+        }
+    };
+    for context in contexts {
+        fixed_columns.push(Column::new(
+            context,
+            false,
+            default_formatter,
+            ColumnSortable::No,
+            Expandable::Disabled,
+            column_formatter,
+        ));
+    }
+    fixed_columns
+}
+
+#[component]
+pub fn compare_overrides() -> impl IntoView {
+    let tenant_rws = use_context::<RwSignal<Tenant>>().unwrap();
+    let org_rws = use_context::<RwSignal<OrganisationId>>().unwrap();
+    let (context_rs, context_ws) = create_signal::<Conditions>(Conditions::default());
+    let (req_inprogess_rs, req_inprogress_ws) = create_signal(false);
+    // this vector stores the list of contexts the user is comparing
+    let contexts_vector_rws = create_rw_signal(vec!["default_config".to_string()]);
+    let source = move || {
+        let tenant = tenant_rws.get().0;
+        let org_id = org_rws.get().0;
+        let contexts = contexts_vector_rws.get();
+        (tenant, org_id, contexts)
+    };
+    let dimension_resource = create_resource(
+        move || (tenant_rws.get().0, org_rws.get().0),
+        |(tenant, org)| async {
+            fetch_dimensions(&PaginationParams::all_entries(), tenant, org)
+                .await
+                .unwrap_or_default()
+        },
+    );
+    let resolved_config_resource =
+        create_blocking_resource(source, |(tenant, org_id, contexts)| async move {
+            let mut contexts_config_vector_map: ComparisonTable = HashMap::new();
+            for context in contexts {
+                match resolve_config(&tenant, &context, &org_id, false).await {
+                    Ok(Value::Object(config)) => {
+                        for (config_key, resolved_value) in config {
+                            if let Some(row_vector) =
+                                contexts_config_vector_map.get_mut(&config_key)
+                            {
+                                row_vector.insert(
+                                    "config_key".into(),
+                                    Value::String(config_key.clone()),
+                                );
+                                row_vector.insert(context.clone(), resolved_value);
+                                continue;
+                            }
+                            let mut row_vector = Map::new();
+                            row_vector.insert(
+                                "config_key".into(),
+                                Value::String(config_key.clone()),
+                            );
+                            row_vector.insert(context.clone(), resolved_value);
+                            contexts_config_vector_map.insert(config_key, row_vector);
```

Comment:
simply doing this would also work, right ?
```suggestion
                            let mut row_vector = contexts_config_vector_map.get_mut(&config_key).unwrap_or_default();
                            row_vector.insert(
                                "config_key".into(),
                                Value::String(config_key.clone()),
                            );
                            row_vector.insert(context.clone(), resolved_value);
                            contexts_config_vector_map.insert(config_key, row_vector);
```

Reply:
in that case this would work, right ?
we just dont use the `Default` trait
```suggestion
                            let mut row_vector = contexts_config_vector_map.get_mut(&config_key).unwrap_or(&Map::new());
                            row_vector.insert(
                                "config_key".into(),
                                Value::String(config_key.clone()),
                            );
                            row_vector.insert(context.clone(), resolved_value);
                            contexts_config_vector_map.insert(config_key, row_vector);
```